evap/staff/importers/enrollment.py

```diff
@@ -608,12 +617,23 @@ def import_enrollments(
             store_participations_in_db(parsed_rows)
 
             msg = format_html(
-                _("Successfully created {} courses/evaluations, {} participants and {} contributors:{}"),
-                new_course_count,
-                new_participants_count,
-                new_responsibles_count,
-                create_user_list_html_string_for_message(new_user_profiles),
+                _("Successfully created {evaluation_string}, {participant_string} and {contributor_string}"),
+                evaluation_string=ngettext(
+                    "one course/evaluation", "{count} courses/evaluations", new_course_count
+                ).format(count=make_count_word(new_course_count)),
+                participant_string=ngettext("one participant", "{count} participants", new_participants_count).format(
+                    count=make_count_word(new_participants_count)
+                ),
+                contributor_string=ngettext("one contributor", "{count} contributors", new_responsibles_count).format(
+                    count=make_count_word(new_responsibles_count)
+                ),
             )
+
+            if new_participants_count or new_responsibles_count:
```

Comment:
I like the idea, but we should also add a "." if there is no list. And we also didn't do this in the other change above